numpy/core/_add_newdocs.py

```diff
@@ -1536,6 +1536,10 @@
 
         .. versionadded:: 1.20.0
 
+    Returns
+    -------
+    out : ndarray
+
```

Comment:
I think this needs to explain what happens if `like` is not `None`

Reply:
... and if this is the docstring for np.ma.frombuffer then it not correct there.